src/components/checkbox-group/checkbox/bl-checkbox.ts

```diff
@@ -0,0 +1,96 @@
+import { CSSResultGroup, html, LitElement, TemplateResult } from 'lit';
+import { customElement, property, query } from 'lit/decorators.js';
+import style from './bl-checkbox.css';
+import type BlCheckboxGroup from '../bl-checkbox-group';
+import { event, EventDispatcher } from '../../../utilities/event';
+
+/**
+ * @tag bl-checkbox
+ * @summary Baklava Checkbox component
+ */
+
+@customElement('bl-checkbox')
+export default class BlCheckbox extends LitElement {
+  static get styles(): CSSResultGroup {
+    return [style];
+  }
+  private checkboxGroup: BlCheckboxGroup | null;
+
+  @query('#checkbox')
+  checkbox: any;
```

Comment:
We don't use `any` you can use more specific `HTMLInputElement`.

Reply:
fixed